.github/workflows/tests.yml

```diff
@@ -42,3 +51,16 @@ jobs:
         run: apt-get update && apt-get install -y jq
       - name: Run tests
         run: ./bin/test-exercises.sh
+  exercises-windows:
+    name: Check exercises Crystal on Windows
+    runs-on: windows-2022
+    steps:
+      - name: Install Crystal
+        uses: crystal-lang/install-crystal@v1
```

Comment:
Could you use a hash to pin the workflow?

Reply:
Could you use this same setup method instead of:

```
container:
      image: crystallang/crystal
```

Reply:
This piece of code is already live. But no windows doesnt support container.

Reply:
I meant the other way around, instead of using the Docker image, use `crystal-lang/install-crystal@v1`

Reply:
Ohh yeah that would work